repositories/user_repository.py

```
@@ -0,0 +1,64 @@
+            self._save_all([])
+
+    #내부용 함수
+    def _load_all(self) -> List[dict]:
+        with open(self.file_path,"r",encoding="utf-8") as f:
+            try:
+                return json.load(f)
+            except json.JSONDecodeError:
+                return []
+
+    #내부용 함수
+    def _save_all(self, users: List[dict]):
```

Ruff (0.14.11)

Missing return type annotation for private function `_save_all`

Add return type annotation: `None`

(ANN202)